File: dataset/output/OPENAI_gpt-4o/Task45_OPENAI_gpt-4o.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter an integer: ");
        try {
            int number = scanner.nextInt();
            System.out.println("The entered integer is: " + number);
        } catch (Exception e) {
            System.out.println("Invalid input. Please enter an integer.");
        }
    }
}

File: dataset/output/OPENAI_gpt-4o/Task45_OPENAI_gpt-4o.py
import java.util.Scanner;

public class MultiLangProgram {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number for Java: ");
        try {
            int javaNum = scanner.nextInt();
            System.out.println("Java read the number: " + javaNum);
        } catch (Exception e) {
            System.out.println("Invalid input in Java: " + e.getMessage());
        }
    }
}